DependencyInjection/EzPlatformUIExtension.php

```diff
@@ -45,6 +45,11 @@ public function prepend(ContainerBuilder $container)
         $this->prependYui($container);
         $this->prependCss($container);
         $this->prependImageVariations($container);
+
+        $configFile = __DIR__ . '/../Resources/config/kernel.yml';
```

Comment:
`views.yml` is still better, I think. Given that the kernel is meant to be split into packages in the near future, referencing the "kernel" will become a bit meaningless. Furthermore, it is what is used as a best practice (demos, tutorials...).